### ISSUE TITLE
Inclusive Date Range

### ISSUE THREAD

**Issue by codingfriend1** [quasarframework/quasar#868]
### Feature
Provide ability for `date.isBetweenDates(date, dateFrom, dateTo)` to be inclusive. Right now it's just exclusive and I'm having to do this if I'm wanting to check if a date belongs in two days.

```
date.isBetweenDates(
  day,
  date.subtractFromDate(dateFrom, {days: 1}),
  date.addToDate(dateTo, {days: 1})
)
```

Perhaps an `includedInRange` method?

```
date.isIncludedInRange(
  day,
  dateFrom
  dateTo
)
```

Or a configuration object.

```
date.isBetweenDates(
  day,
  dateFrom
  dateTo,
  {
    inclusiveFrom: true,
    inclusiveTo: false,
  }
)
```

I'm not to particular on the api, so do what you feel is best.

Thanks! And thanks for all the hard work on an awesome framework!

**Comment by rstoenescu** [comment]
Hi, Please help out with a PR. I'm overloaded with work. Thank you.

**Comment by rstoenescu** [comment]
Will be available in v0.15.3
